src/main/webapp/app/stores/open-comment.store.ts

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash';
+import { action, makeObservable, observable } from 'mobx';
+class CommentsToDeleteStore {
+  commentIndiciesToDelete: number[] = [];
+
+  constructor() {
+    makeObservable(this, {
+      commentIndiciesToDelete: observable,
+      addCommentIndex: action.bound,
+      removeCommentIndex: action.bound,
+      clear: action.bound,
+    });
+  }
+
+  addCommentIndex(index: number) {
+    this.commentIndiciesToDelete.push(index);
+  }
+
+  removeCommentIndex(index: number) {
+    _.remove(this.commentIndiciesToDelete, commentIndex => commentIndex === index);
+  }
+
+  clear() {
+    this.commentIndiciesToDelete = [];
+  }
+}
+
+export const openCommentScrollPosition = observable.box<number>(null);
+export const openCommentsIdStore = observable.box<string>(null);
```

Comment:
Wondering why these are not in the comment store and as constants?

Reply:
FIxed